app/_locales/nl/messages.json

```diff
@@ -0,0 +1,819 @@
+{
+  "accept": {
+    "message": "Aanvaarden"
+  },
+  "account": {
+    "message": "Account"
+  },
+  "accountDetails": {
+    "message": "Accountgegevens"
+  },
+  "accountName": {
+    "message": "Accountnaam"
+  },
+  "address": {
+    "message": "Adres"
+  },
+  "addCustomToken": {
+    "message": "Aangepaste token toevoegen"
+  },
+  "addToken": {
+    "message": "Voeg token toe"
+  },
+  "addTokens": {
+    "message": "Tokens toevoegen"
+  },
+  "amount": {
+    "message": "Bedrag"
+  },
+  "amountPlusGas": {
+    "message": "Bedrag + gas"
+  },
+  "appDescription": {
+    "message": "Ethereum Browser-extensie",
+    "description": "De beschrijving van de applicatie"
+  },
+  "appName": {
+    "message": "MetaMask",
+    "description": "De naam van de applicatie"
+  },
+  "attemptingConnect": {
+    "message": "Poging om verbinding te maken met blockchain."
+  },
+  "attributions": {
+    "message": "Bevoegdheden"
+  },
+  "available": {
+    "message": "Beschikbaar"
+  },
+  "back": {
+    "message": "Terug"
+  },
+  "balance": {
+    "message": "Balans:"
+  },
+  "balances": {
+    "message": "Je saldo"
+  },
+  "balanceIsInsufficientGas": {
+    "message": "Onvoldoende saldo voor huidig ​​gastotaal"
+  },
+  "beta": {
+    "message": "BETA"
+  },
+  "betweenMinAndMax": {
+    "message": "moet groter zijn dan of gelijk zijn aan $ 1 en kleiner dan of gelijk aan $ 2.",
```

Comment:
this is a variable, needs to be `$1` and `$2`

Reply:
Missed these, sorry!